.stylelintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  "extends": [
+    "stylelint-config-standard"
+  ],
+  "customSyntax": "postcss-styled-syntax",
+  "rules": {
+    "no-empty-source": null,
```

Comment:
This allows us to create styled components without any styles in them. We aim not to have many of those, but when an element has repetitive properties, it can be useful.
e.g.
```
const Details = styled.div.attrs<{ isDetailed?: boolean }>(props => ({
  className: props.isDetailed ? grid({ s: 12, m: 6, l: 8, xl: 8 }) : '',
}))<{ isDetailed?: boolean }>``;
```